_topics/en/silent-payments.md

```diff
@@ -73,5 +73,23 @@ excerpt: >
 
 ---
 
+Traditionally, a new address should be generated whenever a user receives
+two or more payments. This is because when you receive several payments
+to the same Bitcoin address, others can reasonably assume that the same
+person received all of those payments even if the payments are later
+spent in separate transactions. This is also known as address reuse,
+and more can be found under [output linking][topic output linking].
+
+With silent payments, a receiver can generate a silent payment address
+and make it publicly known. The sender then takes a public key from
+one of their chosen inputs for the payment, and uses it to derive a
```

Comment:
This part is slightly inaccurate.

Sender can select one <ins>or more</ins> inputs and uses their <ins>secret</ins> keys to derive the shared secret (together with public key of the silent payment address). It is receiver who uses public keys from inputs (and corresponding secret key of the silent payment address).

Reply:
Yes, thank you for the correction! I copied this part from Ruben's original gist [here](https://gist.github.com/RubenSomsen/c43b79517e7cb701ebf77eec6dbb46b8) which is a little outdated as @josibake mentioned. Will update this paragraph to be more accurate 👍